app/models/salt_handler/minion_orchestration.rb

```diff
@@ -12,16 +12,18 @@ def self.can_handle_event?(event)
     parsed_event_data = JSON.parse event.data
     parsed_event_data["fun"] == "runner.state.orchestrate" &&
       (parsed_event_data["fun_args"].first == "orch.kubernetes" ||
-       parsed_event_data["fun_args"].first["mods"] == "orch.kubernetes")
+       parsed_event_data["fun_args"].first == "orch.update" ||
+       parsed_event_data["fun_args"].first["mods"] == "orch.kubernetes" ||
+       parsed_event_data["fun_args"].first["mods"] == "orch.update")
```

Comment:
You could even check whether `data["fun_args"]` is nil beforehand, just to extra-safe 😉